tenant/service_user.go

```diff
@@ -8,6 +8,12 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
+// UserService is a union of UserService and PasswordsService.
+type UserService interface {
+	influxdb.UserService
+	influxdb.PasswordsService
+}
+
```

Comment:
This is confusing. the system already understands the user service https://github.com/influxdata/influxdb/blob/master/user.go#L43-L64 By redefining it here it will become difficult to understand which implementation we are using.